### PR TITLE
Updated text for automation security

### PR DESCRIPTION
**Pull request by SharifChoudhury1**
Text was automated for clarity

**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/test/end-to-end/pages/generateCorrespondence/generateCorrespondence.js [hunk 1]
```diff
@@ -5,10 +5,21 @@ const testConfig = require("../../../config");
...
+    I.see('Top Level');
     I.waitForText(commonConfig.lettersCorrespondence, testConfig.TestTimeToWaitForText);
     I.selectOption('#correspondenceType_topLevel_Documents', commonConfig.lettersCorrespondence);
     I.waitForText(commonConfig.lettersCorrespondence1, testConfig.TestTimeToWaitForText);
     I.selectOption('#correspondenceType_part_2_Documents', commonConfig.lettersCorrespondence1);
+    I.see('Letters');
     I.click(commonConfig.continue);
     I.click(commonConfig.submit)
+    //After submitting letters
+    I.see('Please download the document from : ')
```
Comment (SunnyAyeni) [inline src/test/end-to-end/pages/generateCorrespondence/generateCorrespondence.js:21]: please add a wait statement here. 
after clicking submit button the next page could take some time to load

Comment (SunnyAyeni) [reply src/test/end-to-end/pages/generateCorrespondence/generateCorrespondence.js:21]: before the `I see` step

Comment (SharifChoudhury1) [reply src/test/end-to-end/pages/generateCorrespondence/generateCorrespondence.js:24]: Added a wait statement.

**Review by pats-john — APPROVED** [review]

File: src/test/end-to-end/pages/generateCorrespondence/generateCorrespondence.js (hunk 1 above)
Comment (SharifChoudhury1) [reply src/test/end-to-end/pages/generateCorrespondence/generateCorrespondence.js]: Yes, waited for a header, and then saw the statement

**Review by SunnyAyeni — APPROVED** [review]

**Pull request merged**